include/ppx/application.h

```diff
@@ -548,6 +549,11 @@ class Application
     void   DestroyPlatformWindow();
     bool   IsRunning() const;
 
+    // DispatchConfig() allows derived classes to update settings
+    // This is the final step of updating the settings
+    // based on the values defined in DispatchConfig() and mStandardOpts
+    void FinalizeSettings();
```

Comment:
I feel like this comment doesn't tell me what this function is useful for.
It seems `FinalizeSettings` is applying the knobs options to the app settings.
Maybe `UpdateStandardSettings()` would be a better name?
And for the comment, what about something like:

```cpp
// Updates the standard application settings to reflect the knob values. 
```